### PR TITLE
Kokkos-Kernels,Amesos2,ROL: Fix tiny issues

### PR DESCRIPTION
**Pull request by cgcgcg**
@trilinos/kokkos-kernels @trilinos/amesos2 @trilinos/rol

### CODE REVIEW

File: packages/amesos2/test/solvers/MUMPS_test.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
 	  <Parameter name="LocalOrdinal" type="string" value="int"/>
 	  <Parameter name="GlobalOrdinal" type="strig" value="int"/>
```
Comment (brian-kelley) [inline packages/amesos2/test/solvers/MUMPS_test.xml:32]:
```suggestion
	  <Parameter name="GlobalOrdinal" type="string" value="int"/>
```

**Review by brian-kelley — CHANGES_REQUESTED** [review]

File: packages/amesos2/test/solvers/MUMPS_test.xml (hunk 1 above)
Comment (brian-kelley) [reply packages/amesos2/test/solvers/MUMPS_test.xml:32]: If we're fixing typos :)
Your changes all look fine though

Comment (cgcgcg) [reply packages/amesos2/test/solvers/MUMPS_test.xml]: Dang, good catch!

**Review by brian-kelley — APPROVED** [review]

**Comment by ndellingwood** [comment]
@cgcgcg the kokkos-kernels changes will need to be submitted to the kokkos-kernels repo as well to avoid clobber during the next release

**Comment by brian-kelley** [comment]
@ndellingwood It looks like the KK develop already has the typo fixed by @lucbv

**Comment by cgcgcg** [comment]
I fixed it here because I couldn't find that file in the KK repo, but maybe I didn't look in the right spot.

**Comment by ndellingwood** [comment]
I extracted the kokkos-kernels change to apply to the release-candidate-4.2.01 branch , kokkos/kokkos-kernels@21074411575267f136e47ecb946acc855da72979

**Pull request merged**